### PR TITLE
Bracket standards

### PR DESCRIPTION
**Pull request by JeffMatson**

### CODE REVIEW

File: acpt.php [hunk 1]
```diff
@@ -12,117 +12,112 @@
...
+		printf( '<div class="%1$s"><p>%2$s</p></div>', $class, $message );
 	}
 
-	$Advanced_Custom_Post_Types = new acpt();
+}
 
-endif;
+if ( ! class_exists( 'acpt' ) ) {
+	$Advanced_Custom_Post_Types = new acpt();
+}
```
Comment (iambriansreed) [inline acpt.php]: Wouldn't this never fire? The class has been defined.

File: acpt.php [hunk 2]
```diff
@@ -7,15 +7,14 @@
 Author URI: http://iambrian.com/
 */
 
-if ( ! defined( 'ABSPATH' ) )
```
Comment (iambriansreed) [inline acpt.php:10]: Bracket standards = always use brackets, or alternate. Where does it say not newline?

File: acpt.php [hunk 3]
```diff
@@ -48,7 +48,7 @@ function __construct() {
 
 			// all back end related functionality is only loaded if needed
 			if ( is_admin() ) {
-				require dirname( __FILE__ ) . '/admin.php';
+				require_once dirname( __FILE__ ) . '/admin.php';
```
Comment (iambriansreed) [inline acpt.php]: Agreed.

File: acpt.php [hunk 4]
```diff
@@ -117,13 +117,4 @@ function admin_notice_acf_not_activated() {
...
 	$Advanced_Custom_Post_Types = new acpt();
-}
-
-//dev only
-
-function pre( $value, $exit = false ) {
-	echo '<pre>' . print_r( $value, 1 ) . '</pre>';
-	if ( $exit ) {
-		exit;
-	}
```
Comment (iambriansreed) [inline acpt.php:151]: You got me. Forgot to stash.

File: acpt.php (hunk 2 above)
Comment (JeffMatson) [inline acpt.php:10]: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style

Usually, you want to use a new line.  On short things like this, I'll usually drop it all in a single line though. Preference here for the most part.

**Comment by iambriansreed** [comment]
This PR was out of sync and incorporated most fixes while reviewing issues. Thanks @JeffMatson!

**Pull request closed without merging**